lessons/intro_and_strings/strings.py

```diff
@@ -79,9 +104,13 @@
 # In many programming languages, there is a separate char data type
 # to refer to characters. This data type doesn't exist in Python.
 # So, for example, even this variable declaration:
-# my_char = "A"
+my_char = "A"
 # has a data type that is a string.
-# print(type(my_char)) > str
+print(type(my_char)) # > str
+
+# MH:
+#     is it valid/common to put a comment at the end of a line (as in 109 above?)
```

Comment:
It is common to do this with lines of code you don't want to be linted.

For example:

````
stuff = "This is my super long anti - pep 8 string but i love it so i want it to stay in place exactly like this." # noqa